features/git-town/configuration/non_feature_branches/modify_non_feature_branch_configuration.feature

```diff
@@ -9,15 +9,21 @@ Feature: modifying the non-feature branch configuration
     Given my non-feature branch is "qa"
     And I have a branch named "staging"
     When I run `git town non-feature-branches --add staging`
-    Then I see "Added 'staging' as a non-feature branch"
+    Then I see no output
     And the non-feature branches include "staging"
 
 
   Scenario: adding a non-feature branch that already exists
     Given I have a branch named "qa"
     And my non-feature branch is "qa"
-    When I run `git town non-feature-branches --add qa`
-    Then I see "'qa' is already a non-feature branch"
+    When I run `git town non-feature-branches --add qa` while allowing errors
+    Then I see
+      """
+
+        Error
+        'qa' is already a non-feature branch
```

Comment:
I have never seen error messages formatted like this. Most have the format: `Error: 'qa' is already a non-feature branch`.